mlir/lib/Dialect/SCF/Utils/Utils.cpp

```diff
@@ -1171,3 +1372,10 @@ scf::ForOp mlir::fuseIndependentSiblingForLoops(scf::ForOp target,
 
   return fusedLoop;
 }
+
+scf::ParallelOp mlir::fuseIndependentSiblingParallelLoops(
+    scf::ParallelOp target, scf::ParallelOp source, RewriterBase &rewriter) {
+  auto mayAlias = [&](Value val1, Value val2) -> bool { return false; };
```

Comment:
not exactly sure what to do with this, but i figured if this was intended to be some "naive" with no checks that we can relax any aliasing conditions. feedback on this much appreciated